src/main/java/org/openhab/binding/zwave/handler/ZWaveThingHandler.java

```diff
@@ -423,7 +427,7 @@ public void run() {
                         controllerHandler.sendData(message);
                     }
                 } catch (Exception e) {
-                    logger.warn("NODE {}: Polling aborted due to exception", nodeId, e);
+                    logger.debug("NODE {}: Polling aborted due to exception", nodeId, e);
```

Comment:
Probably this should be WARN - this is quite a serious error.

Reply:
Changed to warn.